source/common/router/rds_impl.h

```diff
@@ -98,6 +103,11 @@ class RdsRouteConfigSubscription : Envoy::Config::SubscriptionCallbacks,
 public:
   ~RdsRouteConfigSubscription() override;
 
+  std::unordered_set<RouteConfigProvider*>& routeConfigProviders() {
+    return route_config_providers_;
+  }
+  RouteConfigUpdatePtr& routeConfigUpdateInfo() { return config_update_info_; }
```

Comment:
Inconsistency here between the type "RouteConfigUpdate" and "routeConfigUpdateInfo".